### PR TITLE
The Whippet, a Bare-Bones Cargo Hauler

### PR DESCRIPTION
**Pull request by Vytian**
## About the PR

Sometimes you want a smaller operation while still being able to haul cargo and haul it fast. Does the loader have too many luxuries that could be taken up by cargo space? This fuselage with thrusters attached is sure to get you and your cargo to its destination nice and quick, with minimal space dedicated to crew amenities to maximize cargo space. Buy a Whippet today!

## Why / Balance

It'd be nice to have some more variety from just seeing the Loader whenever crews of one or two want to run cargo. Costs 17500 spesos to be competitive with the Loader.

## Technical details

Simply adds the Whippet shuttle model to the shipyard.

## Media

Whippet

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

- add: NC Whippet ship model to the shipyard

### CODE REVIEW

File: Resources/Prototypes/_NF/Shipyard/whippet.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  name: NC Whippet
+  description: A no-frills cargo hauler to get large amounts there and back fast.
+  price: 17500
+  category: Small
+  group: Civilian
+  shuttlePath: /Maps/Shuttles/whippet.yml
+
+- type: gameMap
+  id: Whippet
+  mapName: 'NC Whippet'
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/_NF/Shipyard/whippet.yml:12]: • Change mapname to "Whippet" and not "MC Whippet" to avoid long names in IFF.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
• Missing LateJoin Spawn.
• Emercency O2/Captain locked need to use the Filled version and not fill it yourself.

**Review by TsjipTsjip — COMMENTED** [review]
See FoxxyTrystan's notes, additionally I would like to request you look into how your shuttle looks without all those catwalks everywhere in the cargo area. In my opinion, these make your shuttle look very busy.

Did not test this shuttle in-game, but the files look fine aside from already made notes.

**Comment by Vytian** [comment]
Updated with the recommended changes. LateJoin Spawnpoint was already present, just under the cargotechnician spawnpoint instead of over it for some reason.

**Comment by Everturning** [comment]
gonna say it. it looks like a....

inappropriate thing.

**Comment by Everturning** [comment]
move the engines to the center more
add some sort of a drink

**Comment by MagnusCrowe** [comment]
Going to close this for now due to inactivity. Please open it up again when you come back and work on it.

**Pull request closed without merging**